.drone.yml

```diff
@@ -2,6 +2,10 @@ workspace:
   base: /go
 
 pipeline:
+  clone:
+    image: plugins/git
+    tags: true
```

Comment:
`DRONE_TAG` default value is `${DRONE_BRANCH}-${DRONE_COMMIT_SHA:0:7}` if you set clone `tags` as `true`.